source/frontend/src/components/common/form/YesNoSelect.tsx

```diff
@@ -1,4 +1,7 @@
+import './Select.scss';
+
 import cx from 'classnames';
+import classNames from 'classnames';
```

Comment:
We already have a "**classnames**" import here - please remove duplicate import

Reply:
I tried. this was the only way to import it.